packages/block-library/src/search/edit.js

```diff
@@ -73,12 +74,18 @@ export default function SearchEdit( {
 	} = attributes;
 
 	const borderRadius = style?.border?.radius;
+	const borderColor = style?.border?.color;
+	// Grab the borderProps so we have access to computer class names for the border color.
```

Comment:
minor typo, I assume that is `computed class names` ?

Reply:
Oh yes... 




🤓 Let's keep this between you and me.